src/components/Tile/Tile.js

```diff
@@ -1,9 +1,40 @@
 const React = require("react");
+const blockedLinks = require("lib/blockedLinks");
+const {connect} = require("react-redux");
+const actions = require("actions/index");
 
 const Tile = React.createClass({
+  /**
+   * Blocks the site (removes it from the grid) and calls the given callback
+   * when done.
+   */
+  block: function (e) {
+    e.preventDefault();
+    if (!this.isBlocked()) {
+      blockedLinks.block(this.props.url);
+      if (this.props.dispatch) {
+        // Dispatch doesn't exist in a test environment.
```

Comment:
hmm... we really should avoid having code to overcome issues in the test environment (i.e., the tests should be modified instead of this code).